src/api/feed-discovery/README.md

```diff
@@ -5,17 +5,17 @@ The Feed Discovery Service provides an autodiscovery for feed url.
 ## Install
 
 ```
-npm install
+pnpm install
 ```
 
 ## Usage
 
 ```
 # normal mode
-npm start
+pnpm start
 
 # dev mode with automatic restarts
-npm run dev
+pnpm run dev
```

Comment:
Get rid of `run` here (we don't use it with `pnpm`).

Reply:
This hasn't been fixed.  Please remove `run`, it should be `pnpm dev`

Reply:
argh, forgot to stage the changes before amending...